javascript/javascript3/week1/week1.js

```diff
@@ -0,0 +1,9 @@
+fetch('http://api.icndb.com/jokes/random/3')
```

Comment:
In general, it is a good idea to add code for error handling. For example in cases when the requests fails. fetch(url) .then(response) => {} .catch((error) => { console.error('Error:', error); });